src/sonic-config-engine/data/l2switch.j2

```diff
@@ -17,7 +17,8 @@
             {% if keyPort != "admin_status" %}"{{ keyPort }}": "{{ valuePort }}",{% endif %}
             {%- endfor %}
 
-            "admin_status": "{{ value.admin_status|default("up") }}"
+            "admin_status": "{{ value.admin_status|default("up") }}",
+            "speed": "{{ value.speed|default("50000") }}"
```

Comment:
>speed [](http://example.com/codeflow?start=13&length=5)

Not all the ports on all the platforms support 50G speed. It it the responsibility of port_config.ini or hwsku.json to define the init port speed.